lightning/src/chain/keysinterface.rs

```diff
@@ -342,6 +340,25 @@ pub trait KeysInterface: Send + Sync {
 	fn get_channel_id(&self) -> [u8; 32];
 }
 
+#[derive(Clone)]
+/// Holds late-bound channel data.
+/// This data is available after the channel is known to be accepted, either
+/// when receiving an open_channel for an inbound channel or when
+/// receiving accept_channel for an outbound channel.
+struct AcceptedChannelData {
```

Comment:
Ah, yes.  Made this and its fields private, as well as the relevant field in the containing structure.